www/package.json

```diff
@@ -48,7 +48,7 @@
     "gatsby-source-npm-package-search": "^2.0.0",
     "gatsby-transformer-csv": "^2.0.0",
     "gatsby-transformer-documentationjs": "^2.0.0",
-    "gatsby-transformer-remark": "^2.1.8",
```

Comment:
I could have also just pinned this--but may as well standardize while we're at it.